applications/nrf5340_audio/CMakeLists.txt

```diff
@@ -53,6 +53,7 @@ configure_file("${CMAKE_CURRENT_SOURCE_DIR}/src/utils/fw_info_app.c.in"
 target_sources(app PRIVATE
     src/main.c
     ${CMAKE_BINARY_DIR}/fw_info_app.c
+    ${ZEPHYR_NRF_MODULE_DIR}/boards/arm/nrf5340_audio_dk_nrf5340/board_version.c
```

Comment:
This looks ok, but there may be a better way to do this include. Here, the app will have dependencies on the board. So, if one wants to run the app on a standard DK, it will need files from the Audio DK to work.

Reply:
I agree on this. But I guess the application is dependent on board_version.c "out of the box" now (we unconditionally call board_version functions from main now). This puts us in a little pickle if board_version.c/.h have to live inside the boards folder.

On a potential side note: I think it is possible to use ${BOARD_DIR}/board_version.c

Reply:
@alexsven are there any updates on this one?

Reply:
After looking a bit at it I see that we can't change this to BOARD_DIR. This is a specific file, so it should have a specific path.

Reply:
what in those files are board specific in such a way they must be adjusted on a per-board basis ?

I don't see anything regarding those files in the docs:
https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/applications/nrf5340_audio/README.html

and if movinf those files then the users must be made aware of their existence and how / what to adjust.

Without knowing it myself, then it's hard to give recommendations.
Are we perhaps talking about those settings:
https://github.com/nrfconnect/sdk-nrf/blob/22826ab5b7c5b5402064d8e167a945f61b41fcd1/applications/nrf5340_audio/src/utils/board_version.c#L23-L26